index.ts

```diff
@@ -50,30 +48,18 @@ class DAppPeerConnect {
     onApiEject,
     onApiInject,
   }: DAppPeerConnectParameters) {
-
-    this.dAppInfo = dAppInfo
+    this.dAppInfo = dAppInfo;
 
     this.meerkat = new Meerkat({
       seed: seed || localStorage.getItem('meerkat-dapp-seed') || undefined,
       announce: announce,
       loggingEnabled: loggingEnabled,
     });
 
-    if(onConnect) {
-      this.onConnect = onConnect
-    }
-
-    if(onDisconnect) {
-      this.onDisconnect = onDisconnect
-    }
-
-    if(onApiEject) {
-      this.onApiEject = onApiEject
-    }
-
-    if(onApiInject) {
-      this.onApiInject = onApiInject
-    }
+    this.onConnect = onConnect;
```

Comment:
I removed the if statements because there is a check for every this.[function name] anyway in the code below before it gets executed

Reply:
The other changes besides of the toLowerCase() methods have been made by prettier's auto formatting regarding to the .prettierrc.json file in the root dir